src/webgpu/api/operation/rendering/blending.spec.ts

```diff
@@ -279,7 +281,96 @@ g.test('formats')
     `Test blending results works for all formats that support it, and that blending is not applied
   for formats that do not. Blending should be done in linear space for srgb formats.`
   )
-  .unimplemented();
+  .params(u =>
+    u //
+      .combine('format', [
+        'r16float',
+        'bgra8unorm-srgb',
+        'r8unorm',
+        'rg16float',
+        'rgb10a2unorm',
+        'rgba16float',
+        'rgba8unorm',
+      ] as const)
+  )
+  .fn(async t => {
+    const { format } = t.params;
+
+    const pipeline = t.device.createRenderPipeline({
+      layout: 'auto',
+      fragment: {
+        targets: [
+          {
+            format,
+            blend: {
+              color: { srcFactor: 'one', dstFactor: 'one', operation: 'add' },
+              alpha: {},
+            },
+          },
+        ],
+        module: t.device.createShaderModule({
+          code: `
+struct Uniform {
+  color: vec4<f32>
+};
+@group(0) @binding(0) var<uniform> u : Uniform;
```

Comment:
This will require a uniform buffer binding, but if you just output a constant `vec4<f32>(0.4, 0.4, 0.4, 0.4)` then you won't need it.